pkg/migration-tool/planner/slab.go

```diff
@@ -118,6 +144,48 @@ func (s *Slab) Fetch(ctx context.Context, client *utils.Client, mint, maxt int64
 	return nil
 }
 
+func (s *Slab) mergeSubSlabsToSlab(subSlabs []*utils.PrompbResponse) []*prompb.TimeSeries {
+	l := len(subSlabs)
```

Comment:
do we really need this var?

Reply:
I have changed this to `numSubuSlabs` and now it helps in reading. Do you think it's trivial even now?